Lombiq.HelpfulExtensions/Extensions/SiteTexts/Startup.cs

```diff
@@ -0,0 +1,26 @@
+using Lombiq.HelpfulExtensions.Extensions.SiteTexts.Services;
+using Microsoft.Extensions.DependencyInjection;
+using OrchardCore.Data.Migration;
+using OrchardCore.Modules;
+
+namespace Lombiq.HelpfulExtensions.Extensions.SiteTexts;
+
+[Feature(FeatureIds.SiteTexts)]
+public class Startup : StartupBase
+{
+    public override void ConfigureServices(IServiceCollection services)
+    {
+        services.AddScoped<IDataMigration, Migrations>();
+        services.AddScoped<ISiteTextService, SiteTextService>();
+    }
+}
+
+[Feature("OrchardCore.ContentLocalization")]
```

Comment:
Not sure what the exact difference is but in the OC source they usually use the `RequireFeatures` attributes for this purpose.

Reply:
I think I referenced this from FINI, so it must be wrong :joy: